tests/FunctionCallTest.php

```diff
@@ -1818,7 +1818,7 @@ function example($x) : void {
                             if ($c !== 2) {}
                         }
                     }',
-                'error_message' => 'TypeDoesNotContainType',
+                'error_message' => 'DocblockTypeContradiction',
```

Comment:
This change is because `TypeDoesNotContainType` was probably generated in psalm internal hardcoding (which can probably be removed by now, given the new stubs), but now `if ($c !== 2) {}` clashes with the return type of `count()` instead.